### PR TITLE
ShadowUI stylesheets are not restored after the document is cleaned (close #310)

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @inikulin @churkin

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

File: src/client/sandbox/shadow-ui.js [hunk 1]
```diff
@@ -124,6 +124,32 @@ export default class ShadowUI extends SandboxBase {
...
+        var stylesheets = this.nativeMethods.querySelectorAll.call(this.document, 'link.' + SHADOW_UI_CLASS_NAME.uiStylesheet);
+        var result      = '';
+
+        for (var i = 0; i < stylesheets.length; i++)
+            result += stylesheets[i].outerHTML;
+
+        return result;
+    }
+
+    _restoreUIStyleSheets (head, uiStyleSheetsHtml) {
```
Comment (inikulin) [inline src/client/sandbox/shadow-ui.js]: It's always used with the `this.uiStyleSheetsHtml` as the second parameter. So, do we need this parameter?

File: src/client/sandbox/shadow-ui.js [hunk 2]
```diff
@@ -27,6 +26,7 @@ export default class ShadowUI extends SandboxBase {
 
         this.root              = null;
         this.lastActiveElement = null;
+        this.uiStyleSheetsHtml = null;
```
Comment (inikulin) [inline src/client/sandbox/shadow-ui.js]: Maybe it will be better name it `uiStyleSheetsHtmlBackup`?

**Comment by inikulin** [comment]
\r- Just minor issue.

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
ping @churkin

**Comment by churkin** [comment]
lgtm

**Pull request merged**